flink-connector-dynamodb/src/test/java/org/apache/flink/connectors/dynamodb/PackagingITCase.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.flink.connectors.dynamodb;
+
+import org.apache.flink.packaging.PackagingTestUtils;
+import org.apache.flink.test.resources.ResourceTestUtils;
+
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+
+/** Test to verify contents of packaged jar. */
+public class PackagingITCase {
+    @Test
+    void testPackaging() throws Exception {
+        final Path jar = ResourceTestUtils.getResource(".*/flink-connector-dynamodb-[^/]*\\.jar");
+
+        PackagingTestUtils.assertJarContainsOnlyFilesMatching(jar, Arrays.asList("META-INF/"));
```

Comment:
This will break the build for the main code contribution I assume. Can we relax this check to support the intended result? We expect class file etc...

Reply:
I have relax the regex to match any dir name prefix with "flink-"